File: backend/src/main/java/com/urmyfood/backend/domain/model/ShopProfile.java
package com.urmyfood.backend.domain.model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.experimental.SuperBuilder;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@SuperBuilder
public class ShopProfile extends BaseDomainModel {
    private Account shop;
    private String shopName;
    private String logoUrl;
    private String coverUrl;
    private String category;
    private String address;
    private Double latitude;
    private Double longitude;
    private String description;
    private String openingHours;
    private Boolean isOpen;
    private String verificationStatus;
}

File: frontend/app-shop/src/main/java/com/urmyfood/shop/domain/model/ShopProfile.kt
package com.urmyfood.shop.domain.model

data class ShopProfile(
    val id: Long?,
    val shopId: Long?,
    val shopName: String,
    val logoUrl: String?,
    val coverUrl: String?,
    val category: String,
    val address: String,
    val latitude: Double?,
    val longitude: Double?,
    val description: String?,
    val openingHours: String,
    val isOpen: Boolean,
    val verificationStatus: String
)
